glusterd.toml.example

```diff
@@ -0,0 +1,3 @@
+localstatedir = "/var/lib/glusterd"
+rpcaddress = ":24008"
+restaddress = ":24007"
```

Comment:
Yes you can. All command line flags will be available in config files. The config files can have extra options that aren't available as command line flags.

This file is just an example, and doesn't document all the options.